frontend/src/tests/pages/HomePage.test.js

```diff
@@ -10,184 +10,194 @@ import { systemInfoFixtures } from "fixtures/systemInfoFixtures";
-        );
-        expect(screen.getByTestId("HomePage-main-div")).toBeInTheDocument();
-
-        const greetingElement = screen.getByTestId("homePage-title");
-        expect(greetingElement.textContent).toMatch(new RegExp(expectedGreeting, 'i'));
-      });
+  };
+
+  afterEach(() => {
+    global.Date = originalDate;
+    axiosMock.reset();
+  });
+  const testCases = [
+    { hour: 9, expectedGreeting: "Good morning, cgaucho" },
+    { hour: 14, expectedGreeting: "Good afternoon, cgaucho" },
+    { hour: 19, expectedGreeting: "Good evening, cgaucho" },
+    { hour: 12, expectedGreeting: "Good morning, cgaucho" },
+    { hour: 18, expectedGreeting: "Good afternoon, cgaucho" },
+  ];
+
+  testCases.forEach(({ hour, expectedGreeting }) => {
+    test(`shows "${expectedGreeting}" for not logged-in user at ${hour} hours`, () => {
+      mockDateWithHour(hour);
+      axiosMock.onGet("/api/currentUser").reply(200, apiCurrentUserFixtures.userOnly);
+      render(
+        <QueryClientProvider client={queryClient}>
+          <MemoryRouter>
+            <HomePage />
+          </MemoryRouter>
+        </QueryClientProvider>
+      );
+      expect(screen.getByTestId("HomePage-main-div")).toBeInTheDocument();
+
+      const greetingElement = screen.getByTestId("homePage-title");
+      expect(greetingElement.textContent).toMatch(new RegExp(expectedGreeting, 'i'));
     });
   });
+});
 
 
 
-  describe('HomePage greetings for logged-in users at different times of the day', () => {
-    const originalDate = global.Date;
-    const axiosMock = new AxiosMockAdapter(axios);
-    const queryClient = new QueryClient();
+describe('HomePage greetings for logged-in users at different times of the day', () => {
+  const originalDate = global.Date;
+  const axiosMock = new AxiosMockAdapter(axios);
+  const queryClient = new QueryClient();
 
 
-    const mockDateWithHour = (hour) => {
-      global.Date = class extends Date {
+  const mockDateWithHour = (hour) => {
+    global.Date = class extends Date {
 
-        getHours() {
-          return hour;
-        }
-        getDate() { return originalDate.now(); }
-        getMonth() { return originalDate.now(); }
-        getFullYear() { return originalDate.now(); }
-      };
+      getHours() {
+        return hour;
+      }
+      getDate() { return originalDate.now(); }
+      getMonth() { return originalDate.now(); }
+      getFullYear() { return originalDate.now(); }
     };
-  
+  };
 
-    beforeEach(() => {
-        global.Date = originalDate;
-        axiosMock.reset();
-        axiosMock.resetHistory();
-        //axiosMock.onGet("/api/currentUser").reply(200, apiCurrentUserFixtures.adminUser);
-        axiosMock.onGet("/api/systemInfo").reply(200, systemInfoFixtures.showingAll);
 
-    });
+  beforeEach(() => {
+    global.Date = originalDate;
+    axiosMock.reset();
+    axiosMock.resetHistory();
+    //axiosMock.onGet("/api/currentUser").reply(200, apiCurrentUserFixtures.adminUser);
```

Comment:
No commented out code please.  I see that it was there in the original, but if you touch the file, you now own it.

Either remove this or consider restoring the file so that it doesn't show up in the diff.